.evergreen/docker/README.md

```diff
@@ -22,22 +22,59 @@ will be passed through to the container.
 The appropriate port(s) will be exposed, allowing you to run local test against
 the running docker container.
 
-The default image can be overridden with `IMAGE``, and the entrypoint with `ENTRYPOINT`.
+The default image can be overridden with `IMAGE`, and the entrypoint with `ENTRYPOINT`.
 To use a specific architecture, use `PLATFORM`, e.g. `--platform linux/amd64`.
 
-## Driver Testing in Docker
+The script also supports the standard environment variables used in mongo-orchestration:
 
-To extend this image and run against a driver test suite, first build the
-image locally.
+```
+AUTH
+SSL
+TOPOLOGY
+LOAD_BALANCER
+STORAGE_ENGINE
+REQUIRE_API_VERSION
+DISABLE_TEST_COMMANDS
+MONGODB_VERSION
+MONGODB_DOWNLOAD_URL
+ORCHESTRATION_FILE
+```
+
```

Comment:
Suggest adding the following explanation on the default behavior of TOPOLOGY and ORCHESTRATION_FILE:

````suggestion

Note that the default `TOPOLOGY` is [`servers`](https://github.com/mongodb-labs/drivers-evergreen-tools/tree/master/.evergreen/orchestration/configs/servers) and the default `ORCHESTRATION_FILE` is `basic.json`. For example, to run a replica using the [auth](https://github.com/mongodb-labs/drivers-evergreen-tools/blob/master/.evergreen/orchestration/configs/replica_sets/auth.json) file orchestration:

```bash
TOPOLOGY=replica_set ORCHESTRATION_FILE=auth.json bash ./run-server.sh 
```
````